### PR TITLE
Work on computer selector widget

### PR DESCRIPTION
**Pull request by yakutovicha**
1) Set selected_computer and computers be traits.
2) Rename codes.ipynb to codes_computers.ipynb and add computer
selector demonstrator there. Modify start.py accordingly.

### CODE REVIEW

File: aiidalab_widgets_base/computers.py [hunk 1]
```diff
@@ -652,72 +652,86 @@ def append_text(self, append_text):
...
+        to the 'options' trait of `self._dropdown` widget.
+    """
 
-    def __init__(self, text='Select computer:', **kwargs):
-        """ Dropdown for Codes for one input plugin.
+    selected_computer = Union([Unicode(), Instance(Computer)], allow_none=True)
+    computers = Dict(allow_none=True)
+
+    def __init__(self, text='Select computer:', path_to_root='../', **kwargs):
+        """Dropdown for Codes for one input plugin.
```
Comment (ltalirz) [inline aiidalab_widgets_base/computers.py]: please fix

File: aiidalab_widgets_base/computers.py [hunk 2]
```diff
@@ -652,72 +652,86 @@ def append_text(self, append_text):
...
 class ComputerDropdown(ipw.VBox):
-    """Widget to select a configured computer."""
+    """Widget to select a configured computer.
+
+    Attributes:
+        selected_computer(Unicode or Computer): Trait that points to the selected Computer instance.
+            It can be set either using Computer or computer's name. It is linked to the 'value' trait
+            of `self._dropdown` widget.
+
+        computers(Dict): Trait that contains computers found in the AiiDA database. It is linked
```
Comment (ltalirz) [inline aiidalab_widgets_base/computers.py]: Here I would use similar wording to the suggestion for the code dropdown

File: aiidalab_widgets_base/computers.py [hunk 3]
```diff
@@ -652,72 +652,86 @@ def append_text(self, append_text):
...
-            self._dropdown.label = selected_computer
+        if isinstance(computer, str):
+            if computer in self.computers:
+                return self.computers[computer]
+            raise KeyError("The computer named '{}' wasn't found in AiiDA database.".format(computer))
+
+        if isinstance(computer, Computer):
+            if computer.name in self.computers:
+                return computer
+            raise ValueError("The computer '{}' wasn't found in AiiDA database.".format(computer))
```
Comment (ltalirz) [inline aiidalab_widgets_base/computers.py]: also here similar to code widget

File: aiidalab_widgets_base/computers.py (hunk 1 above)
Comment (yakutovicha) [reply aiidalab_widgets_base/computers.py]: done

File: aiidalab_widgets_base/computers.py (hunk 2 above)
Comment (yakutovicha) [reply aiidalab_widgets_base/computers.py]: done

File: aiidalab_widgets_base/computers.py (hunk 3 above)
Comment (yakutovicha) [reply aiidalab_widgets_base/computers.py]: done

**Pull request merged**